R/dplyr_methods.R

```diff
@@ -546,8 +546,10 @@ slice_sample.SingleCellExperiment <- function(.data, ..., n=NULL,
         select(!!c_(.data)$symbol) %>%
         count(!!c_(.data)$symbol)
 
+    .max_cell_count <- ifelse(nrow(count_cells)==0, 0, max(count_cells$n))
+
     # If repeated cells due to replacement
-    if (count_cells$n |> max() |> gt(1)){
+    if (.max_cell_count |> gt(1)){
```

Comment:
looks good, does the warning appear in `tidyseurat` as well?

Reply:
Thank you for checking. Seems to be yes, although it results in an error. Should I also replace the part for `tidyseurat`?

```
> pbmc_small |> slice_sample(n=0)
Error: No cells found
In addition: Warning message:
In max(count_cells$n) : no non-missing arguments to max; returning -Inf
```

Reply:
@noriakis  Yes, please fix the error in `tidyseurat` and write a unit test for it.

If this is relative to an empty object I thought you had a `stop()` call already.

Reply:
Thank you for your reply. The error is related to an empty object coming from `stop()` call in `subset.Seurat()` as you pointed out, so I will just fix the warnings.